reviewkitteh/program.py

```diff
@@ -0,0 +1,63 @@
+"""A program that listens for GitHub pull requests and meows at random people.
+
+This program listens for GitHub pull request events and posts a message to a
+Slack channel when a pull request is opened or reopened. It then polls the
+pull request until it is closed or merged, updating the message with the
+current state of the pull request. Every 15 seconds, it also reads a random
+name from a Google Sheet and pages that person in the Slack channel.
+"""
+
+import os
+import time
+import random
+
+from autokitteh.github import github_client
+from autokitteh.google import google_sheets_client
+from autokitteh.slack import slack_client
+
+
+CHANNEL_ID = os.getenv("CHANNEL_ID")
+ORG_DOMAIN = os.getenv("ORG_DOMAIN")
+SHEET_ID = os.getenv("SHEET_ID")
+
+github = github_client("github_conn")
+googlesheets = google_sheets_client("googlesheets_conn").spreadsheets().values()
+slack = slack_client("slack_conn")
+
+
+def on_github_pull_request(event):
+    """Workflow's entry-point."""
+    if event.data.action not in {"opened", "reopened"}:
+        return
```

Comment:
Why not move this to the a filter in the manifest?

Reply:
Good idea. Done